BoostTestAdapter/BoostTestAdapter.csproj

```diff
@@ -10,7 +10,7 @@
     <OutputType>Library</OutputType>
     <AppDesignerFolder>Properties</AppDesignerFolder>
     <RootNamespace>BoostTestAdapter</RootNamespace>
-    <AssemblyName>BoostTestAdapter</AssemblyName>
+    <AssemblyName>BoostTestAdapter.TestAdapter</AssemblyName>

```

Comment:
Please also change the dll name on line 63 in the FilesToLocalize Include otherwise this will break the localization step of our release build. Other than that the changes look good.